frontend/public/style/_common.scss

```
@@ -48,6 +48,12 @@ dl.co-inline {
     padding-left: $pf-v6-global-gutter--md;
     padding-right: $pf-v6-global-gutter--md;
   }
+  .co-m-pane__body-item--cancel-padding {
```

```suggestion
  &--offset-padding {
```
And move to line 63.